dd-sdk-android/src/test/kotlin/com/datadog/android/rum/assertj/ViewEventAssert.kt

```diff
@@ -207,6 +207,47 @@ internal class ViewEventAssert(actual: ViewEvent) :
         return this
     }
 
+    fun hasUserInfo(expected: ViewEvent.Usr?): ViewEventAssert {
```

Comment:
I'd move this `hasUserInfo` next to the other one (but this is nitpicking I know)

Reply:
but I guess it is next to it already